tensorrt_llm/_torch/pyexecutor/py_executor.py

```diff
@@ -917,6 +917,8 @@ def _prepare_and_schedule_batch(self):
         if self.drafter is not None:
             self.use_spec_decode = self.drafter.should_use_spec_decode(
                 self.active_requests)
+            self.draft_forward_post_needed = self.drafter.needs_draft_forward_post(
+            )
```

Comment:
I don't think we need an extra `needs_draft_forward_post` method. Just have it default to a no-op for all drafters and always call it.